raiden/transfer/channel.py

```diff
@@ -844,6 +844,37 @@ def get_amount_locked(end_state: NettingChannelEndState) -> TokenAmount:
     return total_pending + total_unclaimed + total_unclaimed_onchain
 
 
+def get_batch_unlock_gain(
+        channel_state: NettingChannelState,
+) -> Tuple[TokenAmount, TokenAmount]:
+    """Collect amounts for unlocked/unclaimed locks and onchain unlocked locks.
+    Note: this function does not check expiry, so the values make only sense during settlement.
+
+    Returns:
+        gain_from_partner_locks: locks amount received and unlocked on-chain
+        from_from_our_locks: locks amount which are unlocked or unclaimed
+    """
+    gain_from_partner_locks = sum(
+        unlock.lock.amount
+        for unlock in channel_state.partner_state.secrethashes_to_onchain_unlockedlocks.values()
+    )
+
+    """
+    The current participant will gain from unlocking it's own locks when:
+    - The partner never managed to provide the secret to unlock the locked amount.
+    - The partner provided the secret to claim the locked amount but the current
+      participant node never sent out the unlocked balance proof and the partner
+      did not unlock the lock on-chain.
+    """
+    gain_from_our_locks = sum(
```

Comment:
nitpick for readability: Put the 2 different sums in two different variables with appropriate naming and then add these variables